capstone/fitness/src/main/java/com/google/sps/util/Session.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.util;
+
+import com.google.sps.fit.*;
+
+/* Mock Session. */
+public class Session {
```

Comment:
We should discuss the session class since I am using something similar for the progress storage. For mine, it only stores a timestamp, data, and time that the person ran, but I see that you are generalizing a bit more. Mine uses JSON to save the data, so maybe we can adjust my model to make it fit both purposes.